test/bookmarks.js

```diff
@@ -29,6 +29,15 @@ describe('/bookmarks', () => {
     res.body.should.be.a('object');
   });
 
+  it('should return status code 40 when user is not logged in', async () => {
```

Comment:
Should return status code 40?

Reply:
This is not my code. Nevertheless, I will fix it

Reply:
@el-Joft  Good work so far, but please update line 23 and 32 with appropriate message, it doesn't matter if it is your code because you are the one that pushed it.